src/main/java/com/opentable/jackson/OpenTableJacksonConfiguration.java

```diff
@@ -72,6 +82,9 @@ public ObjectMapper objectMapper() {
         if (enableAfterBurner) {
             mapper.registerModule(afterburnerModule());
         }
+        if (registerModuleBeans && (modules != null)) {
```

Comment:
How can this possibly work

default Iterator<T> iterator() {
        return this.stream().iterator();
    }

    default Stream<T> stream() {
        throw new UnsupportedOperationException("Multi element access not supported");
    }

    default Stream<T> orderedStream() {
        throw new UnsupportedOperationException("Ordered element access not supported");
    }